src/dataMap/metaManager/metaSchema.js

```diff
@@ -2890,7 +2890,7 @@ export default () => {
      * @description
      * Disable or enable the nested rows functionality - displaying nested structures in a two-dimensional data table.
      *
-     * See [quick setup of the Nested rows](https://handsontable.com/docs/demo-nested-rows.html).
+     * See [quick setup of the Nested rows]({@link https://handsontable.com/docs/demo-nested-rows.html}).
```

Comment:
After the changes I've seen this:
![image](https://user-images.githubusercontent.com/571316/91015293-35bee680-e5eb-11ea-90f6-9235781ac77b.png)

The `@link` tag is not necessary here as above `@description` tag switches docs description parser to markdown and the `[](link)` pattern do the job.